packages/cycle-scripts-one-fits-all/scripts/test.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+const path = require('path')
+const spawn = require('cross-spawn')
+const chalk = require('chalk')
+
+const mocha = path.resolve(process.cwd(), 'node_modules', '.bin', 'mocha')
```

Comment:
This could easily break, as you are referencing an executable that we are not sure is going to be present (if you want to use mocha, mocha should be a added as dependency)